### PR TITLE
IT-5749 | Prefix DS HTML class names

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: lib/styles/modifiers/_typography.scss [hunk 1]
```diff
@@ -46,66 +47,51 @@
...
+.-titleM-DEPRECATED {
 	@include titleM-DEPRECATED;
 }
 
-.-titleL {
+.-titleL-DEPRECATED {
 	@include titleL-DEPRECATED;
 }
-
-.-buttonXS {
```
Comment (kvas-damian) [inline lib/styles/modifiers/_typography.scss:97]: not used

File: lib/styles/modifiers/_typography.scss [hunk 2]
```diff
@@ -46,66 +47,51 @@
...
 	@include titleS-DEPRECATED;
 }
 
-.-titleM {
+.-titleM-DEPRECATED {
 	@include titleM-DEPRECATED;
 }
 
-.-titleL {
+.-titleL-DEPRECATED {
```
Comment (kvas-damian) [inline lib/styles/modifiers/_typography.scss:94]: fixes some classess not being renamed in https://github.com/bethinkpl/design-system/commit/d83ce04b3f703e2a3df8560fa92803b2e272c18e

File: lib/styles/modifiers/_shadows.scss [hunk 3]
```diff
@@ -1,7 +0,0 @@
-.-shadowLarge {
```
Comment (kvas-damian) [inline lib/styles/modifiers/_shadows.scss:1]: not used after https://github.com/bethinkpl/wnl-platform/pull/4966/commits/6f98ade1390f7c82ff5ec56f66dc09f278b22277

File: lib/styles/modifiers/_layout.scss [hunk 4]
```diff
@@ -1,12 +0,0 @@
-.-centeredSpread {
```
Comment (kvas-damian) [inline lib/styles/modifiers/_layout.scss:1]: moved to platform https://github.com/bethinkpl/wnl-platform/pull/4966/commits/6f98ade1390f7c82ff5ec56f66dc09f278b22277

File: lib/styles/components/_links.scss [hunk 5]
```diff
@@ -1,21 +0,0 @@
-@import '../settings/colors/tokens';
-
-.a-link {
```
Comment (kvas-damian) [inline lib/styles/components/_links.scss:3]: moved to platform, as we have it there in Storybook https://github.com/bethinkpl/wnl-platform/pull/4966/commits/c397d5eacd6332525d6abde67b1604bb9ac0066d

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**